assistant/assistant.go

```diff
@@ -0,0 +1,148 @@
+package assistant
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strings"
+)
+
+const (
+	assistantPath          = "assistant"
+	pocketSphinxUrl        = "http://localhost:8085/stt"
+	tvIsOnLength           = 64512
+	DeskIsOnLength         = 46840
```

Comment:
warning: exported const DeskIsOnLength should have comment (or a comment on this block) or be unexported (golint)